spec/views/view_helpers_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe "Clearance RSpec view spec configuration" do
```

Comment:
Perhaps `type: :view` on this line would make it easier to figure out exactly what it's testing?